extensions/ql-vscode/src/extension.ts

```diff
@@ -951,7 +951,14 @@ async function activateWithInstalledDistribution(
       variantAnalysisSummary: VariantAnalysisApiResponse,
       token: CancellationToken
     ) => {
-      await variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
+
+      const input = [
+        variantAnalysisManager.queue.add(async () => {
```

Comment:
Is it worth it / better to move this implementation to a new method on `VariantAnalysisManager`? It would be a small function but the benefits would be:
- Keeping `extension.ts` as simple as possible
- Keeping knowledge of `queue` internal
  - Sadly the field still have to be `public` so it can be accessed from the tests, but that's fine

Reply:
I see this is similar to a [suggestion by Koen](https://github.com/github/vscode-codeql/pull/1599/files#r996838279) too